src/Kunstmaan/AdminBundle/Helper/AdminPanel/DefaultAdminPanelAdaptor.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Kunstmaan\AdminBundle\Helper\AdminPanel;
+
+use Symfony\Component\Security\Core\SecurityContextInterface;
+
+class DefaultAdminPanelAdaptor implements AdminPanelAdaptorInterface
```

Comment:
That would make overriding the default behavior a lot harder... Right now you just have to override ```kunstmaan_admin.admin_panel.adaptor``` with your own custom implementation to change default behavior (you can inherit the current behavior from it as all methods are protected or public).